cmake/modules/FindSleef.cmake

```diff
@@ -140,4 +140,8 @@ if(Sleef_FOUND)
         INTERFACE_INCLUDE_DIRECTORIES "${Sleef_INCLUDE_DIR}"
     )
   endif()
+
+  if(NOT TARGET Sleef::sleefdft)
+    message(WARNING "Sleef found at ${Sleef_LIBRARY} does not have sleefdft, this means static builds will not link it with Rubberband! Perhaps you meant to use some other version of Sleef?")
```

Comment:
Should I move the message to the `find_package` invocation in `FindRubberband` or keep it here? I'm a bit torn... (in practice it probably doesn't matter much since we only use Sleef for Rubberband)